src/operations/query/filters/aboveAndBelow.js

```diff
@@ -12,6 +14,13 @@ class FilterByAbove extends BaseFilter {
      * @return {import('mongodb').Filter<import('mongodb').DefaultSchema>|import('mongodb').Filter<import('mongodb').DefaultSchema>[]}
      */
     filterByItem(field, value) {
+        if (field === 'id' && isUuid(value)) {
+            return {
+                [this.fieldMapper.getFieldName(UUID_FIELD_NAME)]: {
+                    $gt: value,
+                }
+            };
+        }
```

Comment:
Instead of duplicating the code at many places, let's send value to the getFieldName() and get the apt field as per logic.